### PR TITLE
Add: values for AWS internal setup [TOOLS-136]

### PR DESCRIPTION
**Pull request by dunaev-dmitry**

### CODE REVIEW

File: tip-wlan/example-values/aws-internal/values.yml [hunk 1]
```diff
@@ -0,0 +1,131 @@
+shared:
+  service:
+    srv-https-annotations: &srv-https-annotations
+      kubernetes.io/ingress.class: alb
+      alb.ingress.kubernetes.io/scheme: internet-facing
```
Comment (brennerm) [inline tip-wlan/example-values/aws-internal/values.yml]: Mmh, think this should be set to `internal` as well, or?

Comment (dunaev-dmitry) [reply tip-wlan/example-values/aws-internal/values.yml]: True, missed that somehow

Comment (dunaev-dmitry) [reply tip-wlan/example-values/aws-internal/values.yml]: Fixed

**Comment by brennerm** [comment]
LGTM :+1:

**Review by brennerm — APPROVED** [review]

**Pull request merged**